http/file_server.ts

```diff
@@ -162,7 +162,7 @@ async function serveDir(
     listEntry.push(
       createDirEntryDisplay(
         info.name,
-        fn,
+        fn.replace(currentDir, ""),
```

Comment:
This is not correct. It should be something like `path.join(fn, info.name)`.

I'm not sure what you mean about windows compatibility. It is compatible.